<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QToolButton" name="addmusic">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="showName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>歌曲名</string>
   </property>
  </widget>
  <widget class="QSlider" name="progress">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="showProgress">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>播放进度</string>
   </property>
  </widget>
  <widget class="QToolButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/previous.png</normaloff>:/images/previous.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="showlrcWidget">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>90</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="playlistWidget">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>90</y>
     <width>191</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="playbackMode">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>循环模式</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/sequential.png</normaloff>:/images/sequential.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
